modules/api/src/main/java/org/apache/ignite/tx/TransactionOptions.java

```diff
@@ -39,10 +39,17 @@ public long timeoutMillis() {
     /**
      * Sets transaction timeout, in milliseconds.
      *
-     * @param timeoutMillis Transaction timeout, in milliseconds.
+     * @param timeoutMillis Transaction timeout, in milliseconds. Cannot be negative; 0 means 'use default timeout'.
+     *     For RO transactions, the default timeout is data availability time configured via ignite.gc.lowWatermark.dataAvailabilityTime
```

Comment:
Could you please check that this javadoc renders in a nice way? I'm pretty sure we need some `<p>` tags

Reply:
I wasn't able to make checkstyle pass using `<p>`, so I used `<ul>`